Octokit/Models/Request/NewPullRequest.cs

```diff
@@ -15,7 +15,8 @@ public class NewPullRequest
         /// <param name="title">The title of the pull request.</param>
         /// <param name="head">The branch (or git ref where your changes are implemented. In other words, the source branch/ref</param>
         /// <param name="baseRef">The base (or git ref) reference you want your changes pulled into. In other words, the target branch/ref</param>
-        public NewPullRequest(string title, string head, string baseRef)
+        /// <param name="maintainerCanModify">Whether maintainers of the base repository can push to <paramref name="head"/> if it is a branch</param>
+        public NewPullRequest(string title, string head, string baseRef, bool maintainerCanModify)
```

Comment:
we only want *mandatory* parameters to be in the constructor, so this can be removed sorry!